util/src/main/scala/com/fortysevendeg/mvessel/util/DatabaseUtils.scala

```diff
@@ -0,0 +1,42 @@
+package com.fortysevendeg.mvessel.util
+
+import java.sql.{SQLException, Connection, Statement}
+import scala.util.{Failure, Success, Try}
+
+object DatabaseUtils {
+
+  object WrapSQLException {
+
+    def apply[D, T](option: Option[D], message: String)(f: D => T) = option match {
+      case Some(db) =>
+        Try(f(db)) match {
```

Comment:
![Codacy](https://www.codacy.com/versioned/images/favicon.png) Issue found: [Default case not specified.](https://www.codacy.com/app/47deg/mvessel/file/848926658/issues/source?bid=2442814&fileBranchId=2811084#l12)